src/Workspaces/Core/Portable/Workspace/Solution/Project.cs

```diff
@@ -372,7 +372,9 @@ private static TextDocument CreateAdditionalDocument(DocumentId documentId, Proj
         }
 
         /// <summary>
-        /// Get the <see cref="Compilation"/> for this project if it is available.
+        /// Tries to get the cached <see cref="Compilation"/> for this project if it is available. In almost all
```

Comment:
Maybe change "if it is available" to "if it has already been computed" (or created)?